src/tiles/storage.rs

```diff
@@ -47,7 +47,7 @@ impl TileStorage {
     /// Gets a tile entity for the given tile position, if:
     /// 1) the tile position lies within the underlying tile map's extents *and*
     /// 2) there is an entity associated with that tile position;
-    /// otherwise it returns `None`.
+    ///    otherwise it returns `None`.
```

Comment:
I think this is supposed to be a second paragraph, not a continuation of the second item

Reply:
Indeed, good catch.